resources/js/components/PlayerList.js

```diff
@@ -1,15 +1,14 @@
 // resources/js/components/PlayerList.js
 
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import Player from './Player';
 
 export default function PlayerList(data) {
-    console.log(data);
-
     const [players, setPlayers] = useState(null);
+    const [playerName, setPlayerName] = useState('');
     
     if(players === null) {
-        fetch("http://localhost:8000/api/club-stats/" + data.teamId, {mode: 'no-cors'})
+        fetch("http://localhost:8000/api/club-stats/" + data.team.abbrev, {mode: 'no-cors'})
```

Comment:
id investigate something like tanstack query to do data fetching. Also if you did it this way you'd want to use a `useEffect`